proto/shentu/bounty/v1/tx.proto

```diff
@@ -30,6 +30,9 @@ service Msg {
 
   // MsgReleaseFinding defines a method for release a finding.
   rpc ReleaseFinding(MsgReleaseFinding) returns (MsgReleaseFindingResponse);
+
+  // terminate a program by program_id
+  rpc TerminateProgram(MsgTerminateProgram) returns (MsgTerminateProgramResponse);
```

Comment:
idk about this naming. Maybe just end program is better?

Reply:
"end program" sound no so 'formidable'. 
I've replaced all "terminate" with "end".